app/views/clients.html

```diff
@@ -32,6 +36,10 @@
                   <td data-title="#{{ $index +1 }}"><span id="counter">{{ $index +1 }}</span></td>
                   <td data-title="ID"><a href="" ng-click="editClient(client)">{{ client.clientID }}</a></td>
                   <td data-title="Name"><a href="" ng-click="editClient(client)">{{ client.name }}</a></td>
+                  <td data-title="Organization">{{ client.organization }}</td>
+                  <td data-title="SoftwareName">{{ client.softwareName }}</td>
+                  <td data-title="Description">{{ client.description }}</td>
+                  <td data-title="Contact">{{ client.contactPerson }} ({{client.contactPersonEmail}})</td>
```

Comment:
For this field the brackets still display if the email is empty. I think it would be better to not display the brackets when there is no email address.